container/sagemaker/python_service.py

```diff
@@ -31,13 +37,9 @@
 class InvocationResource(object):
 
     def __init__(self):
-        if 'TFS_GRPC_PORT' not in os.environ:
-            raise EnvironmentError('GRPC port not set')
-        if 'TFS_REST_PORT' not in os.environ:
-            raise EnvironmentError('REST API port not set.')
-
-        self._tfs_grpc_port = os.getenv('TFS_GRPC_PORT')
-        self._tfs_rest_port = os.getenv('TFS_REST_PORT')
+        self._tfs_default_model_name = os.environ['TFS_DEFAULT_MODEL_NAME']
```

Comment:
Should these have defaults?

Reply:
these do have defaults in `serve.py` so they should always be set when `python_service.py` is run. defaulting them again would probably just hide a configuration error.